locales/nb.schema.json

```
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "For å endre utdrag, må du redigere [blogginnleggene.] (https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```

```suggestion
          "content": "For å endre utdrag, må du redigere [blogginnleggene.](https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```
Not part of this PR, but also a broken link

---

Fixed by @rafal-nedzarek-loc in the Smartling backend